partner_contact_nationality/README.rst

```diff
@@ -1,18 +1,18 @@
 .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
-    :alt: License: AGPL-3
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
 
-Module name
-===========
+===================
+Partner Nationality
+===================
 
-This module was written to extend the functionality of Odoo to support setting
-a nationality for your contacts.
+This module was written to extend the contact management functionality. It
```

Comment:
Hi @richard-willowit thanks for the contribution.
This module was written to extend => This module extends